packages/shared/providers.yaml

```diff
@@ -538,6 +538,24 @@ aws:
             example: com
             pattern: '^[a-z.]+$'
 
+aws-iam:
+    display_name: AWS IAM
```

Comment:
This is a bit narrow. Can we have the aws subdomain be a `connection.config` param? Otherwise we are going to need a provider for each separate aws services that has its own subdomain

Reply:
@TBonnin , I might be missing something here, could you please elaborate further?

Reply:
It is linked to the the discussion about connection_base_url I think. I was asking if we are gonna need to have a new provider alias for each separate aws service or if we could have a generic `AWS Signature` provider or something similar.

Reply:
that would be better I think. What do @khaliqgant and @bodinsamuel think?

Reply:
That works. There doesn't seem to be anything specific to `iam` for the integration regardless.

Reply:
linking this discussion with https://github.com/NangoHQ/nango/pull/2942#pullrequestreview-2416097872
The AWS_SIGN_V4 and WSSE auth mode seems to basically be BASIC with a special header that must be constructed a certain way. What about not adding

Reply:
Agree that this should happen but don't think that should block this PR. Tracking in https://linear.app/nango/issue/EXT-221/refactor-providersyaml-to-have-a-signature-attribute